WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -275,7 +275,8 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
             pageHolder.dateHeader.setVisibility(showDate ? View.VISIBLE : View.GONE);
 
             // no "..." more button when uploading
-            pageHolder.btnMore.setVisibility(PostUploadService.isPostUploading(post) ? View.GONE : View.VISIBLE);
+            pageHolder.btnMore.setVisibility(PostUploadService.isPostUploadingOrQueued(post) ? View.GONE :
+                    View.VISIBLE);
```

Comment:
this is interesting! maybe we can do the same for the trash icon instead of adding the Toast I [suggested above](https://github.com/wordpress-mobile/WordPress-Android/pull/6081/files#r122257062)?